server.js

```diff
@@ -0,0 +1,60 @@
+const express = require('express');
+// const helmet = require('helmet');
+
+const baseRouter = require('./data/helpers/base-router.js');
+
+const server = express();
+
+// global middleware
+
+function userUpperCase(req, res, next) {
+
```

Comment:
it looks like your still missing your middleware here if you added it please make sure to push your changes